engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/rest/flinkRestModel.scala

```diff
@@ -53,7 +53,7 @@ object flinkRestModel {
 
   // NOTE: Flink <1.10 compatibility - JobStatus changed package, so we use String here
   @JsonCodec(decodeOnly = true) case class JobOverview(
-      jid: String,
+      jid: JobID,
```

Comment:
Wouldn't it be better to introduce our own FlinkJobId, wrapping either a String or a UUID, instead of relying on a class from the Flink API? If I skimmed through the PR correctly, do we only need the string representation of the job ID?

Reply:
This Flink's JobId is quite stable value class. TBH IMO we should remove the whole flink client related code and use Flink's official client for rest API (`RestClient`/`RestClusterClient`).